src/caller.jl

```diff
@@ -192,6 +192,18 @@ function create_ring_from_singular_ring(r::libSingular.ring_ptr)
       minpoly = F(libSingular.algExt_GetMinpoly(c, F.ptr))
       basering = N_AlgExtField(libSingular.nCopyCoeff(c), minpoly)
       T = n_algExt
+   elseif libSingular.nCoeff_is_Nemo_Field(c)
+      cf = libSingular.nCopyCoeff(c)
+      data_ptr = libSingular.nGetCoeffData(cf)
+      R = unsafe_pointer_to_objref(data_ptr)
```

Comment:
So where is the matching `pointer_from_objref` performed?

Reply:
Yes, and function N_Field{T}(R::Nemo.Field, cached::Bool=true) where T